test/parallel/test-fs-watch-recursive.js

```diff
@@ -22,6 +22,37 @@ const tmpdir = require('../common/tmpdir');
 const testDir = tmpdir.path;
 tmpdir.refresh();
 
+(async () => {
+  // Watch a folder and update an already existing file in it.
+
+  const rootDirectory = fs.mkdtempSync(testDir + path.sep);
+  const testDirectory = path.join(rootDirectory, 'test-0');
+  fs.mkdirSync(testDirectory);
+
+  const testFile = path.join(testDirectory, 'file-1.txt');
+  fs.writeFileSync(testFile, 'hello');
+
+  const watcher = fs.watch(testDirectory, { recursive: true });
+  let watcherClosed = false;
+  watcher.on('change', common.mustCallAtLeast(function(event, filename) {
```

Comment:
There is no need for `common.mustCallAtLeast` in here.

Reply:
what is the downside of including it?

Reply:
There is no need. Since `watcherClosed` check does the same thing, and checks it on exit.

Reply:
yeah but what is the downside of checking something twice in a test?

Reply:
+1 to always wrap all `assert` in a `common.mustCall` or `common.mustCallAtLeast`, better safe than sorry.

Reply:
If we are doing this change, let's do it for all the tests in this file for consistency.

Reply:
> If we are doing this change, let's do it for all the tests in this file for consistency.

Arguably that's not in the scope of this PR.

Reply:
You're right. I recommend: Let's remove this change, and make these changes in a different pull request.

Reply:
It's not a change, it's an addition (and a welcome one IMO).